gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -111,6 +113,16 @@ def validate(self, parsed_args):
       raise ValueError('Both --infer_headers and --representative_header_file '
                        'are passed! Please double check and choose at most one '
                        'of them.')
+    try:
+      # Gets at most one pattern match result of type `filesystems.MatchResult`.
+      first_match = filesystems.FileSystems.match(
```

Comment:
nit: This functionality also exists in `vep_runner` module, please consider removing that as it is not needed anymore.